core/container/src/main/java/org/wildfly/swarm/container/config/ConfigViewFactory.java

```diff
@@ -104,7 +104,7 @@ public ConfigViewFactory load(String profileName) {
     public void load(String profileName, URL url) throws IOException {
         if (url.getPath().endsWith(".properties")) {
             loadProperties(profileName, url);
-        } else if (url.getPath().endsWith(".yml")) {
+        } else if (url.getPath().endsWith(".yml") || url.getPath().endsWith(".yaml")) {
             loadYaml(profileName, url);
         }
     }
```

Comment:
Just a thought: Shouldn't there at least be a warning log entry here in case neither .properties nor .yml/.yaml has been passed in?